js/options/options.js

```diff
@@ -212,7 +212,8 @@ let Options = (function(){
         Region.populateRegionalSelects();
         
         if (!changelogLoaded) {
-            RequestData.getHttp('changelog.txt', {type: "text/plain"}).then(data => {
+            fetch(chrome.runtime.getURL('changelog.txt')).then(r => r.text())
```

Comment:
Please refactor back to using RequestData

Reply:
Created a new PR tfedor/AugmentedSteam#150 https://github.com/tfedor/AugmentedSteam/commit/45fb1cf600915364438babc221ac62d57032a1f3